Rakefile

```diff
@@ -0,0 +1,9 @@
+task :build do
+  system "gem build stellar-core-backup.gemspec"
+end
+
+task :install => :build do
+#  system "gem install stellar-core-backup-0.0.1.gem"
+  system "sudo gem install stellar-core-backup"
```

Comment:
In preparation for pushing to rubygems.org, we probably want to use `gem install --local stellar-core-backup` during development work to ensure the latest local build is installed rather than a remote version

Reply:
Yes -` system "gem install --user-install stellar-core-backup"`

Reply:
So the `--user-install` just specifies to install in the users home dir instead of the GEM_HOME. I was specifically referring to installing from the local filesystem vs pulling from rubygems.org.